packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -163,19 +220,28 @@ async function downloadSimpleFinTransactions(acctId, since) {
     throw BankSyncError(res.error_type, res.error_code);
   }
 
-  const {
-    transactions: { all },
-    balances,
-    startingBalance,
-  } = res;
-
-  console.log('Response:', res);
+  let retVal = {};
+  if (batchSync) {
+    for (const [accountId, data] of Object.entries(
+      res as SimpleFinBatchSyncResponse,
+    )) {
+      retVal[accountId] = {
+        transactions: data.transactions.all,
```

Comment:
The crash is here.  If an account is in error state, it has different `data`:

<img width="651" alt="image" src="https://github.com/user-attachments/assets/7d358989-eb9a-44f5-95c2-ca1d0709e5e5">